mk/0_initial.mk

```diff
@@ -2,4 +2,14 @@
 # Initial file (included as the first)
 #
 
-PROJECT_DIR := $(shell dirname $(abspath $(firstword $(MAKEFILE_LIST))))
+PROJECT_DIR:=$(shell dirname $(abspath $(firstword $(MAKEFILE_LIST))))
+
+GO_VERSION?=1.18.10
+GOBIN:=$(PROJECT_DIR)/bin
+GO?=$(shell go env GOPATH)/bin/go$(GO_VERSION)
+
+GOLINT?=$(GOBIN)/golangci-lint
+GOFUMPT?=$(GOBIN)/gofumpt
+GOIMPORTS?=$(GOBIN)/goimports
+OAPICODEGEN?=$(GOBIN)/oapi-codegen
+TERN?=$(GOBIN)/tern
```

Comment:
In this file, we define the required Go version. We need to regularly bump this but if we are a bit behind the RH container, it is not a big deal. We will just make sure we all are on the same page.